range_test.go

```diff
@@ -515,3 +516,51 @@ func TestRange_GoString(t *testing.T) {
 func TestError_Error(t *testing.T) {
 	assert.Equal(t, "blah", Error("blah").Error())
 }
+func TestRangeMarshalJSON(t *testing.T) {
+	// Create a sample Range instance
+	startTime := time.Date(2023, time.September, 6, 12, 0, 0, 0, time.UTC)
+	duration := time.Hour * 2
+	r := Range{st: startTime, dur: duration}
+
+	// Marshal the Range to JSON
+	jsonData, err := r.MarshalJSON()
+	if err != nil {
```

Comment:
I would prefer to use the `assert` library for such assertions, as it is used in the other tests